tests/converter_test.php

```diff
@@ -546,4 +546,49 @@ public function test_conversion_record_deduplication() {
         $task->execute();
         $this->assertEquals(2, $DB->count_records('file_conversion'));
     }
+
+    /**
+     * Data provider for test_supports
+     * Supported formats: 'doc', 'docx', 'rtf', 'xls', 'xlsx', 'ppt', 'pptx', 'html', 'odt',
+     * 'ods', 'txt', 'png', 'jpg', 'gif', 'pdf'
+     *
+     * @return array
+     */
+    public function supports_provider() {
+        return [
+            ['doc', true], ['DOC', true],
+            ['docx', true], ['DOCX', true],
+            ['rtf', true], ['RTF', true],
+            ['xls', true], ['XLS', true],
+            ['xlsx', true], ['XLSX', true],
+            ['ppt', true], ['PPT', true],
+            ['pptx', true], ['PPTX', true],
+            ['html', true], ['HTML', true],
+            ['odt', true], ['ODT', true],
+            ['ods', true], ['ODS', true],
+            ['txt', true], ['TXT', true],
+            ['png', true], ['PNG', true],
+            ['jpg', true], ['JPG', true],
+            ['gif', true], ['GIF', true],
+            ['pdf', false], ['PDF', false],
+            ['mp3', false], ['MP3', false],
+            ['mp4', false], ['mp4', false],
+        ];
+    }
+
+    /**
+     * Test supports method of converter class.
+     *
+     * @covers       \fileconverter_librelambda\converter::supports
+     * @dataProvider supports_provider
+     *
+     * @param string $format The format to test.
+     * @param bool $expected The expected result.
+     *
+     */
+    public function test_supports($format, $expected) {
+        // Check supported format.
+        $this->assertEquals($expected, \fileconverter_librelambda\converter::supports($format, 'pdf'));
+        $this->assertEquals($expected, \fileconverter_librelambda\converter::supports($format, 'PDF'));
```

Comment:
Second argument should be `$expected`.

Reply:
In my vendor/phpunit/phpunit/src/Framework/Assert.php

it is     "public static function assertEquals($expected, $actual, string $message = ''): void"